File: src/main/java/ru/sicampus/bootcamp2026/dto/UserDto.java
package ru.sicampus.bootcamp2026.dto;

import jakarta.validation.constraints.NotBlank;
import lombok.Data;

@Data
public class UserDto {
    private Long id;
    private @NotBlank String login;
    private @NotBlank String name;
    private @NotBlank String lastname;
    private String aboutMe;
    private @NotBlank String position;
    private String photoUrl;
}

File: src/main/kotlin/ru/sicampus/bootcamp2026/dto/UserDto.kt
package ru.sicampus.bootcamp2026.dto

import java.time.LocalDateTime

data class UserResponseDto(
    val id: Long,
    val email: String,
    val createdAt: LocalDateTime
)

data class UserUpdateDto(
    val email: String? = null,
    val password: String? = null
)
